app/src/lib/score.ts

```diff
@@ -1,17 +1,70 @@
-import { CurrentGameSubscription } from "generated/graphql"
+import { CurrentGameSubscription, GameStatsQuery } from "generated/graphql"
 import { Team } from "lib/team"
-import { filter, flatMap } from "lodash"
+import { filter, flatten, sum } from "lodash"
 
 export function teamScore(
   team: Team,
-  turns: CurrentGameSubscription["games"][0]["turns"],
+  turns: GameStatsQuery["turns"],
   players: CurrentGameSubscription["games"][0]["players"]
 ) {
   const teamPlayerIds = filter(players, (player) => player.team === team).map(
     (player) => player.id
   )
-  const teamTurns = filter(turns, (turn) =>
+  const completedTurns = filter(turns, (turn) => !!turn.review_started_at)
+  const teamTurns = filter(completedTurns, (turn) =>
     teamPlayerIds.includes(turn.player_id)
   )
-  return flatMap(teamTurns, (turn) => turn.completed_card_ids).length
+  const teamScorings = flatten(teamTurns.map((turn) => turn.scorings))
+  return sum(teamScorings.map((turn_scoring) => turn_scoring.score))
+}
+
+type PlayerScore = {
+  [key: string]: number
+}
+
+export function gameStats(
+  turns: GameStatsQuery["turns"],
+  players: CurrentGameSubscription["games"][0]["players"]
+) {
+  const playerScores: PlayerScore = {}
+  players.forEach((player) => {
+    const playerScorings = flatten(
+      filter(turns, (turn) => turn.player_id === player.id).map(
+        (turn) => turn.scorings
+      )
+    )
+    playerScores[player.id] = sum(
+      playerScorings.map((scoring) => scoring.score)
+    )
+  })
+
+  let highScore = -1
+  for (const [, playerScore] of Object.entries(playerScores)) {
+    if (playerScore > highScore) {
+      highScore = playerScore
+    }
+  }
+
+  const highScorePlayers = filter(
+    players,
+    (player) => playerScores[player.id] === highScore
+  )
+
+  const redScore = teamScore(Team.Red, turns, players)
+  const blueScore = teamScore(Team.Blue, turns, players)
+  const teamScores = {
+    [Team.Blue]: blueScore,
+    [Team.Red]: redScore,
+  }
+  const tie = redScore === blueScore
+  const winningTeam = redScore > blueScore ? Team.Red : Team.Blue
+
+  return {
+    playerScores,
+    highScore,
+    highScorePlayers,
+    teamScores,
+    winningTeam,
+    tie,
```

Comment:
we can add more keys here like...

teamScoresByRound
playerScoresByRound
mostDifficultCard
mostDifficultCardByRound

mostSkips
leastSkips

--- other ideas

who submitted which word